CHANGELOG

```diff
@@ -1,3 +1,6 @@
+* Tue Aug 2 2022 igomura <52306882+igomura@users.noreply.github.com> - 5.5.1
```

Comment:
I no not think this is super important, but for author line you can use: Igor Muratov <igomura@gmail.com>

Reply:
Updated, thanks!